CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+### Changed
+- Update check pager_team to override client pager_team setting 
```

Comment:
This needs to be called out as a breaking change. Also this is not really clear as to its intention. I would do something like this:
```
### Breaking Change
- changed the precedence of pager_team evaluation from `client -> check -> json_config`  to `check -> client -> json_config`  
```

This makes it much clearer for someone evaluating CHANGELOG to see the breaking change and its impact.